app/components/wds-community-header.js

```diff
@@ -1,11 +1,31 @@
 import Component from '@ember/component';
 import {computed} from '@ember/object';
+import {htmlSafe} from '@ember/string';
+import { assert } from '@ember/debug';
+import track from '../utils/track';
 
 export default Component.extend({
 	attributeBindings: ['style'],
 	classNames: 'wds-community-header',
 	tagName: 'header',
-	style: computed('model', function () {
-		return `background-image: url(${this.get('model')['background-image-url']});`
+
+	init() {
+		this._super(...arguments);
+
+		assert('Required `track` function for wds-global-footer component is not set', this.track);
```

Comment:
What if we don't want to track those events? I don't think it's needed in announcements. IMO it should be optional.